lib/contracts/contributor.js

```diff
@@ -34,6 +34,29 @@ class Contributor extends Base {
       });
   }
 
+  filterByAccount(search) {
+    return this._byAccount(search, 'filter');
+  }
+
+  findByAccount(search) {
+    return this._byAccount(search, 'find');
+  }
+
+  _byAccount(search, method = 'filter') {
+    return this.all().then((contributors) => {
+      const searchEntries = Object.entries(search);
+
+      return contributors[method]((contributor) => {
+        if (!contributor.accounts) { return false; }
+        return contributor.accounts.find((account) => {
+          return searchEntries.reduce((accumulator, searchValue) => {
```

Comment:
I would replace the `reduce` with an `every` [Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)

```js
searchEntries.every((item) => {
  const [ key, value ] = item;
  return account[key] === value;
});
```

Reply:
oh, reads nice. thanks.